src/dmd/backend/cod3.d

```diff
@@ -2595,21 +2595,74 @@ code *genmovreg(uint to,uint from)
 
 void genmovreg(ref CodeBuilder cdb,uint to,uint from)
 {
-    debug
-        if (to > ES || from > ES)
-            printf("genmovreg(to = %d, from = %d)\n",to,from);
+    genmovreg(cdb, to, from, TYMAX);
+}
+
+void genmovreg(ref CodeBuilder cdb, uint to, uint from, tym_t tym)
+{
+    // register kind. ex: GPR,XMM,SEG
+    static uint _K(uint reg)
+    {
+        switch (reg)
+        {
+        case ES:                   return ES;
+        case XMM15:
```

Comment:
????

Reply:
https://github.com/dlang/dmd/blob/27097d85732c28954679153dba2965bfbbc37a34/src/dmd/backend/code_x86.d#L59-L61
https://github.com/dlang/dmd/blob/27097d85732c28954679153dba2965bfbbc37a34/src/dmd/backend/code_x86.d#L68
https://github.com/dlang/dmd/blob/27097d85732c28954679153dba2965bfbbc37a34/src/dmd/backend/code_x86.d#L76-L79
between `XMM7` and `XMM15` are non `XMM` registers.